File: src/main/java/com/domclick/dto/request/AccountDepositRequest.java
package com.domclick.dto.request;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.PositiveOrZero;
import java.io.Serializable;
import java.math.BigDecimal;

public class AccountDepositRequest implements Serializable {
    @NotNull(message = "Field 'accountId' is required")
    @PositiveOrZero(message = "Field 'accountId' value must be positive")
    private Long accountId;
    @NotNull(message = "Field 'value' is required")
    @PositiveOrZero(message = "Field 'value' value must be positive")
    private BigDecimal value;

    public Long getAccountId() {
        return accountId;
    }

    public void setAccountId(Long accountId) {
        this.accountId = accountId;
    }

    public BigDecimal getValue() {
        return value;
    }

    public void setValue(BigDecimal value) {
        this.value = value;
    }
}

File: app/src/main/kotlin/com/domclick/dto/request/AccountDepositRequest.kt
package com.domclick.dto.request

import javax.validation.constraints.NotNull
import javax.validation.constraints.PositiveOrZero
import java.io.Serializable
import java.math.BigDecimal

class AccountDepositRequest : Serializable {
    @NotNull(message = "Field 'accountId' is required")
    @PositiveOrZero(message = "Field 'accountId' value must be positive")
    var accountId: Long = 0
    @NotNull(message = "Field 'value' is required")
    @PositiveOrZero(message = "Field 'value' value must be positive")
    lateinit var value: BigDecimal
}
